config/config.go

```diff
@@ -38,14 +39,15 @@ var (
 
 // Config defines the top level configuration for a spacemesh node
 type Config struct {
-	BaseConfig `mapstructure:"main"`
-	P2P        p2pConfig.Config      `mapstructure:"p2p"`
-	API        apiConfig.Config      `mapstructure:"api"`
-	HARE       hareConfig.Config     `mapstructure:"hare"`
-	TIME       timeConfig.TimeConfig `mapstructure:"time"`
-	GAS        state.GasConfig       `mapstructure:"gas"`
-	REWARD     mesh.Config           `mapstructure:"reward"`
-	POST       postConfig.Config     `mapstructure:"post"`
+	BaseConfig  `mapstructure:"main"`
+	P2P         p2pConfig.Config      `mapstructure:"p2p"`
+	API         apiConfig.Config      `mapstructure:"api"`
+	HARE        hareConfig.Config     `mapstructure:"hare"`
+	ELIGIBILITY eligConfig.Config     `mapstructure:"eligibility"`
```

Comment:
Why not put it under HARE? Anyway, we have two types of eligibility, one for the hare committee and another for the layer committee. Either make it general for both and rename the specific fields to something with hare or change the topic

Reply:
Using HareEligibility instead.